packages/main/src/Table.js

```diff
@@ -496,6 +517,84 @@ class Table extends UI5Element {
 		}
 	}
 
+	_onkeydown(event) {
+		if (isCtrlA(event)) {
+			event.preventDefault();
+			this.isMultiSelect && this._selectAll(event);
+			return;
+		}
+
+		const isAltUp = isUpAlt(event);
+
+		if (isAltUp || isDownAlt(event)) {
+			return this._handleArrowAlt(isAltUp, event.target);
+		}
+	}
+
+	/**
+	 * Handles Alt + Up/Down.
+	 * Switches focus between column header, last focused item, and "More" button (if applicable).
+	 * @private
+	 * @param {boolean} shouldMoveUp Whether to move focus upward
+	 * @param {object} focusedElement The element currently in focus
+	 */
+	_handleArrowAlt(shouldMoveUp, focusedElement) {
```

Comment:
I think this method can be further optimised. 

May be use if ... else... instead only ifs, as the return in the middle looks strange. 

looking it from another angle, the code may look like this:

```
   if (shouldMoveUp) {
			switch(focusedElementType) {
				case "tableRow" :
					this._previousItem = focusedElement;
					this._onColumnHeaderClick();
					break;
				case "columnHeader" :
					!!moreButton ? moreButton.focus() : this._previousItem.focus();
					break;
				case "moreButton" :
					!!this._previousItem ? this._previousItem.focus() : this._onColumnHeaderClick();
					break;
			}
		} else {
			switch(focusedElementType) {
				case "tableRow" :
					this._previousItem = focusedElement;
					!!moreButton ? moreButton.focus() : this._onColumnHeaderClick();
					break;
				case "columnHeader" :
					if (this._previousItem) {
						this._previousItem.focus();
					} else if (moreButton) {
						moreButton.focus();
					}
					break;
				case "moreButton" :
					this._onColumnHeaderClick();
					break
			}
		}

```

Reply:
This suggestion will not work as our ESLint rules limit indentation to a maximum of 4 tabs. Let me know if splitting the functions seems okay to you:

```js
	_handleUpAlt(focusedElement) {
		const focusedElementType = this.getFocusedElementType(focusedElement);

		switch (focusedElementType) {
			case "tableRow":
				this._previousItem = focusedElement;
				this._onColumnHeaderClick();
				break;
			case "columnHeader":
				const moreButton = this.getMoreButton();
				moreButton ? moreButton.focus() : this._previousItem.focus();
				break;
			case "moreButton":
				this._previousItem ? this._previousItem.focus() : this._onColumnHeaderClick();
				break;
		}
	}

	_handleDownAlt(focusedElement) {
		const focusedElementType = this.getFocusedElementType(focusedElement);

		switch (focusedElementType) {
			case "tableRow":
				this._previousItem = focusedElement;
				!!moreButton ? moreButton.focus() : this._onColumnHeaderClick();
				break;
			case "columnHeader":
				if (this._previousItem) {
					this._previousItem.focus();
				} else if (moreButton) {
					moreButton.focus();
				}
				break;
			case "moreButton":
				this._onColumnHeaderClick();
				break;
		}
	}
```

Another option would be to use if-else statements in the current method.

Reply:
This is the closest I could get to the above suggestion without breaking ESLint rules:

```js
	_handleArrowAlt(shouldMoveUp, focusedElement) {
		const focusedElementType = this.getFocusedElementType(focusedElement);
		const moreButton = this.getMoreButton();

		if (shouldMoveUp) {
			switch (focusedElementType) {
			case "tableRow":
				this._previousItem = focusedElement;
				return this._onColumnHeaderClick();
			case "columnHeader":
				return moreButton ? moreButton.focus() : this._previousItem.focus();
			case "moreButton":
				return this._previousItem ? this._previousItem.focus() : this._onColumnHeaderClick();
			}
		} else {
			switch (focusedElementType) {
			case "tableRow":
				this._previousItem = focusedElement;
				return moreButton ? moreButton.focus() : this._onColumnHeaderClick();
			case "columnHeader":
				if (this._previousItem) {
					return this._previousItem.focus();
				}

				if (moreButton) {
					return moreButton.focus();
				}

				return;
			case "moreButton":
				return this._onColumnHeaderClick();
			}
		}
	}
```

Reply:
Im ok with that